dash_agent/lib/data/objects/web_data_object.dart

```diff
@@ -82,22 +84,29 @@ abstract class WebDataObject {
 /// Example:
 /// ```dart
 /// final webUrl = 'https://sampleurl.com';
-/// final webPageObject = WebDataObject.fromWebPage(webUrl);
+/// final webPageObject = WebDataObject.fromWebPage(webUrl, performDeepCrawl = false);
```

Comment:
let's keep the variable `deepCrawl` only

Reply:
Updated the code based on the suggestion